package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@api3/chains",
   "license": "MIT",
-  "version": "4.1.1",
+  "version": "4.1.2",
```

Comment:
Would you mind setting this back to 4.1.1 please? There's [another PR](https://github.com/api3dao/chains/pull/138) I'd like to include and then I can release v4.1.2 (makes it easier to find things in the commit history)

Reply:
I think adding providers require a minor release and I published that because it was blocking @bdrhn9

Reply:
> I think adding providers require a minor release

Yeah that's probably more correct